### ISSUE TITLE
NTR: Palmitoyl-protein thioesterase activity low in leukocytes or fibroblasts (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8140]
New term request Palmitoyl-protein thioesterase activity low in leukocytes or fibroblasts (forsyt_070802081742): PPT1

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000783
- label: Reduced tissue palmitoyl-protein thioesterase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:9535296
- def: Activity of the enzyme palmitoyl-protein thioesterase (PPT; EC 3.1.2.22) in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts and leukocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8140
- synonyms:
